react/components/InfoCard/index.js

```diff
@@ -25,10 +25,10 @@ import { SanitizedHTML } from '../SanitizedHTML'
 
 const ALLOWED_TAGS = ['p', 'span', 'a', 'div', 'br']
 const ALLOWED_ATTRS = {
-  a: ['class', 'href', 'title', 'target'],
-  span: ['class'],
-  p: ['class'],
-  div: ['class'],
+  a: ['class', 'href', 'title', 'target', 'style'],
+  span: ['class', 'style'],
+  p: ['class', 'style'],
+  div: ['class', 'style'],
```

Comment:
I don't think we should be adding `style` to the allowed attributes. Just `class` should be enough :)

Reply:
@victorhmp The client can use inline styling so no need to have code-level changes. So I have included style tag.

Reply:
I understood your idea, but we don't want to include that for everyone. Using the inline `style` like that would go against our idea of how developers should be styling their stores. By using just `class`, you can create a CSS class and style it in an actual CSS file, not inline.

Reply:
@victorhmp Removed `style` attribute